extensions/amp-story/1.0/amp-story-bling-link.css

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-story-grid-layer amp-story-bling-link {
+  margin: 10px !important;
+  background: rgba(255, 255, 255, 1) !important;
+  border-radius: 50% !important;
+  text-align: center !important;
+  line-height: 75px !important;
+  width: 75px !important;
```

Comment:
Is `75px` the real size we're going to use?

Reply:
It doesn't have to be, just eyeballing. Waiting for UI specs